pkg/micropython/Makefile.dep

```diff
@@ -0,0 +1,11 @@
+USEMODULE += xtimer
+USEMODULE += stdin
+
+# MicroPython doesn't compile for <32bit platforms
+FEATURES_BLACKLIST += arch_8bit arch_16bit
+
+# This port currently requires ISR_STACKSIZE and thread_isr_stack_start
+FEATURES_BLACKLIST += arch_arm7 arch_esp32 arch_esp8266 arch_mips arch_riscv
+
+# The port currently doesn't compile for mips
```

Comment:
reason ?

Reply:
multiple different errors, in addition to ISR_STACKSIZE missing